test/testcafe/spec/ChallengeCustomAppPush_spec.js

```diff
@@ -141,23 +163,32 @@ test
     const logoClass = challengeCustomAppPushPageObject.getBeaconClass();
     const logoBgImage = challengeCustomAppPushPageObject.getBeaconBgImage();
     await t.expect(pushBtn.textContent).contains('Push notification sent');
-    await t.expect(a11ySpan.textContent).contains('Push notification sent');
-    await t.expect(pushBtn.hasClass('link-button-disabled')).ok();
+    if (!userVariables.v3) {
+      await t.expect(a11ySpan.textContent).contains('Push notification sent');
+    }
+    await t.expect(challengeCustomAppPushPageObject.isPushButtonDisabled()).eql(true);
     await t.expect(logoClass).contains('custom-app-logo');
-    await t.expect(logoBgImage).match(/^url\(".*\/img\/icons\/mfa\/customPushLogo\.svg"\)$/);
+    if(userVariables.v3) {
+      await t.expect(logoBgImage).match(/.*\/img\/icons\/mfa\/customPushLogo\.svg$/);
```

Comment:
Gen 3 uses `img` element with `src` attribute while Gen 2 uses a `background-image` style property and url